shopapi/forms.py

```diff
@@ -19,7 +19,3 @@ class Meta:
                   'payment_type', 'shipping_type']
 
 
-class ProductTypeForm(forms.ModelForm):
```

Comment:
@vasyltsar why did you remove these lines?Is it a side effect of not doing pull into master before creating new branch?